corim/signedcorim.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package corim
+
+import (
+	"crypto"
+	"crypto/rand"
+	"errors"
+	"fmt"
+
+	cose "github.com/veraison/go-cose"
+)
+
+var (
+	ContentType = "application/rim+cbor"
+)
+
+// SignedCorim encodes a signed-corim type (i.e., a COSE Sign1 wrapped CoRIM)
+// with signature and verification methods
+type SignedCorim struct {
+	UnsignedCorim UnsignedCorim
+	message       *cose.Sign1Message
```

Comment:
message can be re-worded to  SignCorimMsg ?

Reply:
not sure this proposal would substantially increase clarity